portfolio/src/main/webapp/about-you.html

```diff
@@ -34,5 +35,24 @@ <h1>Fun Facts About <em>You</em></h1>
       </form>
       <div id="user-facts-container"></div>
     </div>
+    <hr class="separator">
+    <h1 class="heading">Musical Favorites</h1>
+    <div class="chart-intro chart-content">
+      <p>As I mentioned in the Hobbies page, I love listening and sharing new music and I would love to provide you 
+        with some tunes that I think you would enjoy based on your favorite genre of music. After selecting your favorite one 
+        below, I will show you 5 songs from that genre that I enjoy and I hope you will as well! Check out the chart on the right to 
+        see if your favorite genre is the most popular🎵🎧</p>
+      <form action="/music-data" method="POST">
+        <label for="fav-genre">Which genre is the closest to your favorite?</label><br>
+        <input type="radio" name="fav-genre" value="Pop"> Pop<br>
+        <input type="radio" name="fav-genre" value="Rock"> Rock<br>
+        <input type="radio" name="fav-genre" value="Rap"> Rap<br>
+        <input type="radio" name="fav-genre" value="R&amp;B"> R&amp;B<br>
+        <input type="radio" name="fav-genre" value="EDM"> EDM<br>
```

Comment:
nit: Is the space before each of the genres needed?

Reply:
I wanted to add some space between the button and the text so that's why there's a space but I just moved the spacing aspect to the css file!